frontend/src/index.js

```diff
@@ -4,12 +4,15 @@ import "antd/dist/antd.min.css";
 import "./assets/styles/main.scss";
 import App from "./App";
 import reportWebVitals from "./reportWebVitals";
+import { BrowserRouter } from "react-router-dom";
 
 const root = ReactDOM.createRoot(document.getElementById("root"));
 root.render(
-    <React.StrictMode>
-        <App />
-    </React.StrictMode>
+    // <React.StrictMode>
+        <BrowserRouter>
+            <App />
+        </BrowserRouter>
+    // </React.StrictMode>
```

Comment:
Please uncomment line no 11 and 15
```suggestion
    <React.StrictMode>
        <BrowserRouter>
            <App />
        </BrowserRouter>
    </React.StrictMode>
```